docs/source/building.rst

```diff
@@ -167,10 +167,36 @@ Regression Manager
 
 .. envvar:: COVERAGE
 
-    Enable to report Python coverage data. For some simulators, this will also report :term:`HDL` coverage.
+    Enable to collect Python coverage data for user code.
+    For some simulators, this will also report :term:`HDL` coverage.
+    If :envvar:`COVERAGE_RCFILE` is not set,
+    branch coverage is collected
+    and files in the cocotb package directory are excluded.
 
     This needs the :mod:`coverage` Python module to be installed.
 
+.. envvar:: COVERAGE_RCFILE
+
+    Location of a configuration file for coverage collection of Python user code
+    using the the :mod:`coverage` module.
+    See https://coverage.readthedocs.io/en/latest/config.html for documentation of this file.
+
+    If this environment variable is set,
+    cocotb will *not* apply its own default coverage collection settings,
+    like enabling branch coverage and excluding files in the cocotb package directory.
+
+    .. versionadded:: 1.7
+
+.. envvar:: COCOTB_LIBRARY_COVERAGE
+
+    Enable to collect Python coverage data for code in cocotb's own library.
+    Branch coverage is collected
+    and only files in the cocotb package directory are included.
+
+    This needs the :mod:`coverage` Python module to be installed.
+
+    .. versionadded:: 1.4
+
```

Comment:
Do we need to document this? Would users be interested in this?

Reply:
I removed it now.